android/expoview/src/main/java/host/exp/exponent/ExpoUpdatesAppLoader.java

```diff
@@ -465,12 +465,16 @@ private boolean isValidSdkVersion(String sdkVersion) {
   private ManifestException formatExceptionForIncompatibleSdk(String sdkVersion) {
     JSONObject errorJson = new JSONObject();
     try {
-      errorJson.put("errorCode", "EXPERIENCE_SDK_VERSION_OUTDATED");
       errorJson.put("message", "Invalid SDK version");
-      errorJson.put("metadata", new JSONObject().put(
-        "availableSDKVersions",
-        new JSONArray().put(sdkVersion))
-      );
+      if (ABIVersion.toNumber(sdkVersion) > ABIVersion.toNumber(Constants.SDK_VERSIONS_LIST.get(0))) {
```

Comment:
can we depend on `Constants.SDK_VERSIONS_LIST` to be ordered and always have the most recent sdk version at index 0?

Reply:
I noticed during QA that android's sdk versions list is in no way ordered. We should update `expo-constants` for that if we need it.

Reply:
I assumed we could since the previous code was doing that (except assuming that the _last_ element was the oldest SDK version), but you're right it looks like even though `SDK_VERSIONS_LIST` is a List, it's initialized with a Set so we can't rely on that here. 

I don't really see any reason it _shouldn't_ be initialized from a list though, so I can change that

Reply:
^ fixed in https://github.com/expo/expo/pull/11184/commits/2436ec1032f7cc0f2a908012d455e00fa3e19e95